supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala

```diff
@@ -27,17 +33,9 @@ object ActionUtils {
     *
     * FYI additional older versions are available on DockerHub at:
     * https://hub.docker.com/r/google/cloud-sdk
-    *
-    * When updating this value, also consider updating the CromwellImagesSizeRoundedUpInGB below.
     */
   val CloudSdkImage: String =
-    config.getOrElse("cloud-sdk-image-url", "gcr.io/google.com/cloudsdktool/cloud-sdk:354.0.0-alpine")
-
-  /*
-   * At the moment, cloud-sdk (584MB for 354.0.0-alpine) and stedolan/jq (182MB) decompressed ~= 0.8 GB
-   */
-  val CromwellImagesSizeRoundedUpInGB: Int =
-    config.getOrElse("cloud-sdk-image-size-gb", 1)
+    config.getOrElse("cloud-sdk-image-url", "gcr.io/google.com/cloudsdktool/cloud-sdk:404.0.0-alpine")
```

Comment:
I'm planning to experiment with the `slim` image at the suggestion of https://partnerissuetracker.corp.google.com/issues/247814958#comment6. I'm planning to run the experiment with `354.0.0-slim` and `404.0.0-alpine`, and possibly `404.0.0-slim` depending on the result of the other experiments.

Reply:
Interesting, good luck with that issue!